packages/angular_devkit/build_angular/test/browser/differential_loading_spec_large.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
```

Comment:
I'd like to add a couple of more test here.

1) Watch mode works as expected
3) Can recover from errors
4) Would also like to verify the contents on the es5 and es2015, Example having a `function` and `class` respectively,
5) Verify that web workers also get differential loaded.

Reply:
Note to myself: Also check for serviceworker

Reply:
I might need help with the web worker test.

Reply:
Just ping with with any question.